src/transformStateWithClones.js

```diff
@@ -7,7 +7,45 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = {
+    ...state,
+  };
+
+  const newActions = [];
```

Comment:
I think history is better name for this variable
```suggestion
  const history = [];
```